docs/source/quickstart/script.md

```diff
@@ -0,0 +1,262 @@
+|                               |                 |                 | Keras model. NOTE: This requires that the          |
+|                               |                 |                 | SavedModel's architecture, loss, metrics, etc are  |
+|                               |                 |                 | the same as the RankingModelIf that is not the     |
+|                               |                 |                 | case, then you can still use a SavedModel from a   |
+|                               |                 |                 | model_file for inference/evaluation only           |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --use_all_fields_at_inference | <class 'bool'>  | False           | Whether to require all fields in the serving       |
+|                               |                 |                 | signature of the SavedModel. If set to False, only |
+|                               |                 |                 | requires fields with required_only=True            |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --pad_sequence_at_inference   | <class 'bool'>  | False           | Whether to pad sequence at inference time. Used to |
+|                               |                 |                 | define the TFRecord serving signature in the       |
+|                               |                 |                 | SavedModel                                         |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --output_name                 | <class 'str'>   | relevance_score | Name of the output node of the model               |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --early_stopping_patience     | <class 'int'>   | 2               | How many epochs to wait before early stopping on   |
+|                               |                 |                 | metric degradation                                 |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --file_handler                | <class 'str'>   | local           | String specifying the file handler to be used.     |
+|                               |                 |                 | Should be one of FileHandler keys in               |
+|                               |                 |                 | ml4ir/base/config/keys.py                          |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --initialize_layers_dict      | <class 'str'>   | {}              | Dictionary of pretrained layers to be loaded.The   |
```

Comment:
why is the type not `dict` ?

Reply:
We read string from command line and parse it into dict.